### PR TITLE
Fixes Issue #325; Improved Readme installation instructions and gener…

### PR DESCRIPTION
**Pull request by Preeticp**
…al clean up.

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
LGTM

File: README.rst [hunk 1]
```diff
@@ -7,90 +7,85 @@ What is the Atomic Developer Bundle (ADB)?
 ==========================================
 
 The Atomic Developer Bundle (ADB) is a prepackaged development
-environment filled with production-grade pre-configured tools that makes
-container developer's lives easier.  The ADB supports the development
+environment filled with production-grade, pre-configured tools, that makes
+container developers' lives easier.  The ADB supports the development
```
Comment (brgnepal) [inline README.rst:11]: Why this is not like `developer's` as you have used this pattern in "What is the Typical Usage Pattern?".

File: README.rst [hunk 2]
```diff
@@ -7,90 +7,85 @@ What is the Atomic Developer Bundle (ADB)?
...
 
 A developer begins using the ADB, in most cases, once they have a
 working application that has been decomposed into micro-services.
 They then follow this general outline of steps:
 
-1. Build a Dockerfile for each microservice that is not able to be used
-   from another source.  In other words, build containers for the parts
-   of the application that are unique and reuse for those that aren't
-   (i.e. a database service)
+1. Consider ways to divide your application into its component services or micro-services. For standard pieces, such as web servers, consider using prebuilt containers from trusted sources. For truly unique pieces, build a custom container.
```
Comment (brgnepal) [inline README.rst]: I think `prebuilt` should be `pre-built`

File: README.rst [hunk 3]
```diff
@@ -7,90 +7,85 @@ What is the Atomic Developer Bundle (ADB)?
...
 
 The Atomic Developer Bundle (ADB) is a prepackaged development
-environment filled with production-grade pre-configured tools that makes
-container developer's lives easier.  The ADB supports the development
+environment filled with production-grade, pre-configured tools, that makes
+container developers' lives easier.  The ADB supports the development
 of multi-container applications against different technologies and
 orchestrators while providing a path that promotes best practices.
 
 As a container developer, you want to use the ADB for these reasons:
```
Comment (brgnepal) [inline README.rst:15]: Could we use `following` instead of `these`?

File: README.rst [hunk 4]
```diff
@@ -7,90 +7,85 @@ What is the Atomic Developer Bundle (ADB)?
...
 
 * Pre-Configured: You don't have to spend time building an environment
```
Comment (brgnepal) [inline README.rst]: Could you make point title as bold? 
i.e. `Pre-Configured`, `Multiple Environment Support` etc

These point's titles are not visible when looking that section as whole.

File: README.rst [hunk 5]
```diff
@@ -113,57 +109,78 @@ application specification, `nulecule`_ for multi-container applications.
...
-   ``sudo dnf install vagrant``
+2. `Install Vagrant`_ for your operating system.
 
 3. Install the `vagrant-service-manager`_ Vagrant plugin:
 
    ``vagrant plugin install vagrant-service-manager``
 
-4. Download the latest ADB from `atlas.hashicorp.com`_.  This can be
-   done automatically by Vagrant:
+4. Download the customized Vagrantfiles provided by the ADB project. These Vagrantfiles will download the ADB and automatically set up provider specific container development environments. They are listed below and more details are available, in the `Using Custom Vagrantfiles for Specific Use Cases`_ section of the `Using the Atomic Developer Bundle`_ document and the `Installation document`_.
```
Comment (brgnepal) [inline README.rst]: Extra comma in 3rd line of point 4 after `available` word.

File: README.rst [hunk 6]
```diff
@@ -113,57 +109,78 @@ application specification, `nulecule`_ for multi-container applications.
...
+      * For `OpenShift Origin Vagrantfile`_ use::
 
-   ``$ sudo vagrant up``
+        $ curl -sL https://raw.githubusercontent.com/projectatomic/adb-atomic-developer-bundle/master/components/centos/centos-openshift-setup/Vagrantfile > Vagrantfile
 
-6. If you will be using host based tools, setup your environment with `vagrant-service-manager`
+      * For `Apache Mesos Marathon Vagrantfile`_ use::
 
-   ``$sudo vagrant service-manager --help``
+        $curl -sL https://raw.githubusercontent.com/projectatomic/adb-atomic-developer-bundle/master/components/centos/centos-mesos-marathon-singlenode-setup/Vagrantfile > Vagrantfile
```
Comment (brgnepal) [inline README.rst]: Need space in between `$` and `curl`

**Comment by brgnepal** [comment]
@Preeticp LGTM with few suggestions.

File: README.rst (hunk 1 above)
Comment (Preeticp) [inline README.rst:11]: Possessive noun grammar rules are different for singular and plural nouns Budhram.

Comment (brgnepal) [inline README.rst:11]: Ok

**Comment by brgnepal** [comment]
LGTM :+1:

**Pull request merged**